source/auth/auth.rst

```diff
@@ -1152,6 +1158,8 @@ Version History
 
 Version 1.8.2 Changes
     * Added MONGODB-IAM auth mechanism
+    * Clarify that drivers should raise and error when a connection string
```

Comment:
I don't think this belongs under a previous version heading. Since you bumped the version to 1.9.0, there should probably be a new "Version 1.9.0 Changes" heading here.

Side note: I don't understand by introduction of MONGODB-IAM itself didn't warrant a minor version bump to 1.9.0 on its own, but that's a separate issue from https://github.com/mongodb/specifications/pull/697. I'm won't push that we correct that, but I did want to make a note of it for @bazile-clyde and @jyemin.

Reply:
I have revised the version history to include the new heading.

Reply:
I am not sure whether this hunk conflicts with changes elsewhere, but it looks like this is a copy of the 1.9.0 changelog and so thus hunk should be removed and just the paragraph under 1.9.0 left over.

Reply:
I have removed this text.

Reply:
It still appears to be in the diff. Also please merge/rebase this PR on master so that it is an accurate representation of what will ultimately be committed.

Reply:
I'm not sure how this change did not get pushed. The text is no longer in the diff.